doc/tutorial/statistical_inference/unsupervised_learning.rst

```diff
@@ -120,28 +96,27 @@ algorithms. The simplest clustering algorithm is :ref:`k_means`.
     	>>> face_compressed = np.choose(labels, values)
     	>>> face_compressed.shape = face.shape
 
-    .. list-table::
-      :class: centered
-
-      *
-        - |face|
 
-        - |face_compressed|
+    .. figure:: /auto_examples/cluster/images/sphx_glr_plot_face_compress_001.png
+       :target: ../../auto_examples/cluster/plot_face_compress.html
 
-        - |face_regular|
+       **Raw image**
```

Comment:
same here, I'm not sure we need to update this part.

Reply:
same comment as above